manual/new-starters.md

```diff
@@ -0,0 +1,50 @@
+#git-secrets
+
+[git-secrets](https://github.com/awslabs/git-secrets) is used to prevent passwords/AWS keys/IDs etc from being committed to a git repository and compromising security.
+
+##Installation
+git-secrets must be placed in PATH so that git picks it up when running git-secrets commands.
+Your PATH is an environment variable that holds directories/folders with programs that can be executed within the command line.
+Usually each of these has their own defined behaviours and execution settings. PATH is sometimes referred to as `bin`.
+
+####MacOS
+Use [Homebrew](https://brew.sh/) to install with the command `brew install git-secrets`. Once installed, quit the terminal application and then re-open.
+Homebrew downloads sometimes take a while, especially if you have not used Homebrew in a while as it will update Homebrew first.
+####Linux
+Clone the git-secrets [repository](https://github.com/awslabs/git-secrets) to your preferred location, then cd into the main directory for the git-secrets repository you have cloned.
+Run `sudo make install`.
+This should place git-secrets into your PATH, but if not you may need to move it and run `sudo make install` again OR include the folder made by running `sudo make install` into your PATH.
+
+Once installed, you can use `git-secrets` or `git secrets` interchangeably within the command line
+
+##Initialising git-secrets on a repository
```

Comment:
Did you find any way to initialise git-secrets on all repositories at once? It might be the .git-template instructions here: https://github.com/awslabs/git-secrets#advanced-configuration

Reply:
ah yes! I must have forgotten to add it, will change the instructions! :ok_hand: